pkg/virtualKubelet/storage/test/mockCacheReader.go

```diff
@@ -0,0 +1,62 @@
+package test
```

Comment:
we already have a mocked cache manager under `/pkg/virtualKubelet/storage/test/mockCacheManager.go`